Makefile

```
@@ -25,3 +25,11 @@ run:
 
 dev: btcpay-appsettings build run
 
+test:
```

checkmake (0.2.2)

[warning] Target body for "test" exceeds allowed length of 5 (6).

(maxbodylength)